pages/common/mtr.md

```diff
@@ -17,3 +17,7 @@
 - Force IP IPv4 or IPV6:
 
 `mtr -4 {{host}}`
+
+- Define the time to wait before sending another packet to the same hop:
```

Comment:
Change this to `Wait for a given time in seconds before sending another packet to the same hop:`